src/main/java/com/gmail/axelwerst/Computer.java

```diff
@@ -0,0 +1,70 @@
+package com.gmail.axelwerst;
+
+public class Computer {
+    public Computer(){
+    }
+    public Computer(int ram, int rom, int CPU, operationSystem SYS, String name) {
+        this.Ram = ram;
+        this.Rom = rom;
+        this.CPU = CPU;
+        this.SYS = SYS;
+        this.Name = name;
+    }
+    int Ram;
+        int Rom;
+        int CPU;
+        operationSystem SYS;
+        String Name;
+
```

Comment:
class variables should be `private`

Reply:
please follow the naming conventions!
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html